juju.bash

```diff
@@ -75,10 +75,7 @@ function juju::deploy
          --force \
          --channel "$JUJU_DEPLOY_CHANNEL" "$JUJU_DEPLOY_BUNDLE"
 
-    ret=$?
-    if (( ret > 0 )); then
-        juju::deploy-failure $ret
-    fi
+    juju::deploy-report $?
```

Comment:
this re-introduces previous behavior deleted [here](https://github.com/charmed-kubernetes/jenkins/pull/1131/files#diff-1c740f8e2163fd9b9ae38b82dbe468314d7384fb48606010c0a89d9833c3f024L96-L99)

Reply:
always REPORT if the deployment succeeded or not...